xbmc/guilib/guiinfo/AddonsGUIInfo.cpp

```diff
@@ -165,7 +165,7 @@ bool CAddonsGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
     case SYSTEM_HAS_ADDON:
     {
       ADDON::AddonPtr addon;
-      value = CServiceBroker::GetAddonMgr().GetAddon(info.GetData3(), addon) && addon;
+      value = CServiceBroker::GetAddonMgr().GetAddon(info.GetData3(), addon, ADDON::ADDON_UNKNOWN, false) && addon;
```

Comment:
Better to use then the available function for them:
```*.cpp
value = CServiceBroker::GetAddonMgr().IsAddonInstalled(info.GetData3());
```
Is more clean and do currently on Addon Manager the same as GetAddon here.

See https://github.com/xbmc/xbmc/blob/master/xbmc/addons/AddonManager.h#L196

Reply:
ok, will update the pr as you suggested